{
    "id": "problem_86",
    "description": "\n\nThe problem is a RecursionError caused by a frozenset. When a program attempts to iterate through a frozenset, the program runs into a recursive loop and results in a RecursionError. The program is unable to break out of the recursive loop and therefore crashes.",
    "output": "{1, 2, 3, 4, 5}\nTraceback (most recent call last):\n  File \"<stdin>\", line 8, in <module>\nNameError: name 'fro' is not defined",
    "tags": [
        "RecursionError",
        "frozenset",
        "Hard"
    ]
}